test/test_sparse_semi_structured.py

```diff
@@ -335,7 +333,7 @@ def test_mm_sparse_second_NT(self, dense_input_shape, dtype, device, backend):
 
         # Currently we don't support int matmul on GPU, so evaluate on CPU and copy over
```

Comment:
maybe now there is torch._int_mm on GPU as well: https://github.com/pytorch/pytorch/issues/107671?

Reply:
PR is not touching this part of the test; not sure about `_int_mm()`, will defer it for @jcaip.